annotations/kubernetes-annotations/src/main/java/io/dekorate/kubernetes/config/DefaultKubernetesConfigGenerator.java

```diff
@@ -29,7 +28,6 @@ public class DefaultKubernetesConfigGenerator implements KubernetesConfigGenerat
   public DefaultKubernetesConfigGenerator(ConfigurationRegistry configurationRegistry) {
     this.configurationRegistry = configurationRegistry;
     this.configurationRegistry.add(new ApplyProjectInfo(getProject()));
-    this.configurationRegistry.add(new ApplyImagePullSecretConfiguration());
```

Comment:
Why does this configurator deisappear?

Reply:
It is no longer needed. Ever since we added automatic mapping of system properties to config objects.